frontend/src/credit_transfers/components/CreditTransferFormDetails.js

```diff
@@ -27,21 +27,7 @@ class CreditTransferFormDetails extends Component {
           <span>
             {`${this.props.fields.initiator && this.props.fields.initiator.name} proposes to `}
           </span>
-          {this.props.loggedInUser.organization.actionsTypeDisplay === 'Buy And Sell' &&
-            <div className="form-group">
-              <select
-                className="form-control"
-                id="proposal-type"
-                name="tradeType"
-                value={this.props.fields.tradeType.id}
-                onChange={this.props.handleInputChange}
-              >
-                <option value="" />
-                <option value="1">Sell</option>
-                <option value="2">Buy</option>
-              </select>
-            </div>
-          }
+          <span> sell </span>
           {this.props.loggedInUser.organization.actionsTypeDisplay !== 'Buy And Sell' &&
             <span> sell </span>
```

Comment:
Is this <span> sell </span> now duplicated or redundant?

Reply:
its redundant. by default it does sell transaction.

Reply:
I mean this line here:  {this.props.loggedInUser.organization.actionsTypeDisplay !== 'Buy And Sell' &&
will that ever happen? Then we would see "sell sell"?

Reply:
oh. yes. if organization is Inactive then it will happen and we could see "sell sell". i will fix it since in main release we still want to have supplier to allow sell credit if they are inactive.